id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -180,38 +184,40 @@ public boolean supports(Class<?> clazz) {
 	 */
 	@Override
 	public void validate(@Nonnull Object target, Errors errors) {
+		boolean isDraftRequest = servletRequest.getRequestURI().contains("Draft");
```

Comment:
Is this a safe approach? Anybody changing the path or uri in the controller will break this part of the code.

Reply:
@ramesh-n Updated code with a safe approach